Sources/TuistCache/GraphMappers/CacheTreeShakingGraphMapper.swift

```diff
@@ -75,21 +75,26 @@ public final class CacheTreeShakingGraphMapper: GraphMapping {
 
     fileprivate func treeShake(schemes: [Scheme], sourceTargets: Set<TargetReference>) -> [Scheme] {
         schemes.compactMap { scheme -> Scheme? in
-            let buildActionTargets = scheme.buildAction?.targets.filter { sourceTargets.contains($0) } ?? []
+            var focusedScheme = scheme
 
-            // The scheme contains no buildable targets so we don't include it.
-            if buildActionTargets.isEmpty { return nil }
+            var focusedBuildAction = scheme.buildAction
```

Comment:
Similar comments in regard to the terminology. I wouldn't prefix those variables with `focused..`